test/input.txt

```diff
@@ -106,11 +106,21 @@
   # find multiple with some keywords
   find Charlie Betsy
 
+
+##########################################################
+# test sort persons command
+##########################################################
+
+  # should show new list of persons which is now sorted alphabetically
+  sort
+
+
```

Comment:
It is recommended to use blank lines sparingly as too many blank lines will increase the height of the code, which increases the amount of scrolling.